python/cudf_polars/cudf_polars/utils/dtypes.py

```diff
@@ -84,6 +87,71 @@ def from_polars(dtype: pl.DataType) -> plc.DataType:
         # TODO: Hopefully
         return plc.DataType(plc.TypeId.EMPTY)
     elif isinstance(dtype, pl.List):
+        # TODO: This doesn't consider the value type.
         return plc.DataType(plc.TypeId.LIST)
     else:
         raise NotImplementedError(f"{dtype=} conversion not supported")
+
+
+@cache
+def to_arrow(dtype: plc.DataType) -> pa.DataType:
+    """
+    Convert a pylibcudf datatype to a pyarrow one.
+
+    Parameters
+    ----------
+    dtype
+        Pylibcudf dtype to convert
+
+    Returns
+    -------
+    Matching pyarrow type.
+
+    Raises
+    ------
+    NotImplementedError for unsupported conversions.
+    """
+    type_id = dtype.id()
+    if type_id == plc.types.TypeId.BOOL8:
```

Comment:
We could save a couple lines here by using a dictionary.

Reply:
If (when) pylibcudf gets type stubs, a dictionary would defeat the type checker determining that this is a total function, which I would prefer to keep. Though could be convinced.

However, as per Brandon's comment above, if pylibcudf offers this function in its interop module then I can delete this code entirely, saving even more lines.